pkg/features/features.go

```diff
@@ -158,6 +158,12 @@ const (
 	// owner: @rfranzke
 	// alpha: v1.43.0
 	ShootMaxTokenExpirationValidation featuregate.Feature = "ShootMaxTokenExpirationValidation"
+
+	// HAControlPlanes allows shoot control planes to be run in High Availability mode.
+	// Note: not all control plane components are run in HA mode
+	// owner: @shreyas-s-rao @timuthy
+	// alpha: v1.45.0
+	HAControlPlanes featuregate.Feature = "HAControlPlanes"
```

Comment:
Please add to https://github.com/gardener/gardener/blob/master/docs/deployment/feature_gates.md as well